.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: java
```

Comment:
we don't use Travis at all. We can remove it.

Reply:
Fixed 22ef989e35d08408dc00e5aef18389dfff15b8cd

Reply:
LGTM